app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    result = command.split(" ")
+    if result[0] == "cp":
```

Comment:
what if files will have the same name ?
![image](https://user-images.githubusercontent.com/103201010/220114480-21d5140e-8aee-40a7-a191-08a3ffd12e7a.png)